torch/testing/_internal/common_cuda.py

```diff
@@ -23,6 +23,44 @@
 if TEST_CUDA:
     torch.ones(1).cuda()  # has_magma shows up after cuda is initialized
     TEST_MAGMA = torch.cuda.has_magma
+    dev_name = torch.cuda.get_device_name(torch.cuda.current_device()).lower()
+    check_names = ['orin', 'concord', 'galen', 'xavier', 'nano', 'jetson', 'tegra']
+    IS_JETSON = False
+    for name in check_names:
+        if dev_name in check_names:
+            IS_JETSON = True
```

Comment:
so confusing lol

Reply:
just curious, what about this block is confusing?

I inserted this comment to better explain:
```
    dev_name = torch.cuda.get_device_name(torch.cuda.current_device()).lower()
    # check if the cuda device name matches jetson naming
    check_names = ['orin', 'concord', 'galen', 'xavier', 'nano', 'jetson', 'tegra']
    IS_JETSON = False
```

Reply:
just curious, what about this block is confusing?

I inserted this comment to better explain:
```

    dev_name = torch.cuda.get_device_name(torch.cuda.current_device()).lower()
    # check if the cuda device name matches jetson naming
    check_names = ['orin', 'concord', 'galen', 'xavier', 'nano', 'jetson', 'tegra']
    IS_JETSON = False
```